src/EditorFeatures/CSharpTest/Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests.csproj

```diff
@@ -29,6 +29,7 @@
     <ProjectReference Include="..\DiagnosticsTestUtilities\Microsoft.CodeAnalysis.EditorFeatures.DiagnosticsTests.Utilities.csproj" />
     <ProjectReference Include="..\TestUtilities\Roslyn.Services.Test.Utilities.csproj" />
     <ProjectReference Include="..\..\Compilers\VisualBasic\Portable\Microsoft.CodeAnalysis.VisualBasic.vbproj" />
+    <ProjectReference Include="..\Test\Microsoft.CodeAnalysis.EditorFeatures.UnitTests.csproj" />
```

Comment:
📝 Normally we would move the necessary helpers into a test utilities project rather than add a direct reference

Reply:
is that something you would like me to try here?  or should i proceed as is?

Reply:
This seems fine. The simplified names aren't bad.